### PR TITLE
Setup staging for props

### PR DESCRIPTION
**Pull request by rindek**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -44,6 +44,7 @@ gem 'thin'
 gem 'uglifier'
 gem 'whenever', require: false
 gem 'pundit'
+gem 'crono'
```
Comment (lubieniebieski) [inline Gemfile]: sort them alphabetically please

File: config/cronotab.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    class CreateSubscriptionsJob
+      def perform
+        Rake::Task["email_digests:create_subscriptions"].execute
+      end
+    end
+  end
+end
+
+## copied from schedule.rb
+Crono.perform(CronTasks::EmailDigests::SendDailyJob).every 1.week, on: :monday, at: "17:00"
```
Comment (lubieniebieski) [inline config/cronotab.rb]: @hubertjakubiak will you please improve this? do we have to create 5 separate crono lines for mon-fri actions?

File: config/environments/staging.rb [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  # NGINX, varnish or squid.
+  # config.action_dispatch.rack_cache = true
+
+  # Disable Rails's static asset server (Apache or NGINX will already do this).
+  config.serve_static_files = true
+
+  # Compress JavaScripts and CSS.
+
+  # config.assets.css_compressor = :sass
+  config.secret_key_base = ENV['SECRET_KEY_BASE']
```
Comment (lubieniebieski) [inline config/environments/staging.rb]: Shouldn't we use `AppConfig` here just to be consistent?

File: deployment/Capfile [hunk 4]
```diff
@@ -0,0 +1,9 @@
+require 'capistrano/setup'
+require 'capistrano/deploy'
+require "capistrano/scm/git"
```
Comment (lubieniebieski) [inline deployment/Capfile]: please be consistent about `"`

File: config/cronotab.rb (hunk 2 above)
Comment (rindek) [inline config/cronotab.rb]: Yes we have, that's how this gem works

File: config/environments/staging.rb (hunk 3 above)
Comment (rindek) [inline config/environments/staging.rb]: This is a copypasta from production.rb, so it's consistent with that env

**Pull request merged**